content/blog/2021/announcing-pycon-india.md

```diff
@@ -5,6 +5,18 @@ Category: 2021
 Slug: pycon-india-announcement
 Summary: Announcing the return of PyCon India as an online conference for 2021.
 
+** *There have been some updates to the conference structure. The updated dates for the conference are __September 17th to 20th__*
```

Comment:
@Sc0rpi0n101 What are these `**` for?
Also, would mentioning the year help?

Reply:
This is just to indicate this is a note, like `** Note: `

I don't think mentioning the year would help much, as just above the line is a massive title with PyCon India 2021.